proposals/wac-ucr/index.bs

```diff
@@ -717,6 +727,32 @@ This allows Erin to work with new measurements for the `Acme` trial without
 being exposed to measurements from other trials, or already processed
 measurements from the `Acme` trial.
 
+### Conditional access by relationship ### {#conditional-relationship}
+
+Felicia is responsible for preparing and sharing internal reports for
+senior research team members. These reports include
+direct links to source
+data, including individual measurement [=resources=] in the `measurements`
+[=collection=].
+
+Senior research team members do not have regular access to `measurements`.
+They only access individual [=resources=] in `measurements` that are linked
+through these reports. Each report only links to a small subset of individual
+measurements. Reports are often amended to reference additional measurements
+over time.
+
+When Felicia prepares a new report `/reports/report-1`, she authorizes only
+the senior research team members receiving the report to have
+inherited [=read access=] to
+`measurements`, with a condition that the [=resources=] must:
+
+* **include** a link from `/reports/report-1`
+    with a link type of `hasMeasurement`
+
```

Comment:
I think in the use case less detail may fit better while proposal satisfying requirements motivated by the use case would need to include more details. Use case could only specify

* "with a condition that measurements must be related to report-1 with `ex:hasMeasurement` predicate.


Later proposal could include details that to satisfy this use case one could:

* have `/reports/report-1#this` report where `/reports/report-1` document would also include statements about which measurements were used to generate that report. Those statements would have to match pattern `<reports/report-1/#this> ex:hasMeasurement ?measurement . `. it would also need to address situations where we would have document `</measurements/2020-08-13>` including statements describing multiple measurements `</measurements/2020-08-13#13:40:00.173Z>` `</measurements/2020-08-13#13:41:00.337Z>` etc. where the report only *has* subset of them.

Reply:
Agree on the specifics you mentioned should be included in follow-on proposal @elf-pavlik 👍   In the meantime I believe the requested change has been reflected in the latest commit.